packages/vscode-extension/src/loggers/VSConsoleLogger.ts

```diff
@@ -63,8 +63,7 @@ export default class VSConsoleLogger extends ConsoleLogger {
 	// TODO: SdkExecutor.js should reject in a structured way that contains error codes
 	// This logic could be moved to the catch block receiving that reject object
 	private checkForCorruptedJar(message: string) {
-		
-		if (message.includes(INVALID_JAR_FILE_MESSAGE)) {
+		if (typeof message?.includes === 'function' && message.includes(INVALID_JAR_FILE_MESSAGE)) {
```

Comment:
This logger could be triggered from node-cli javasscript code and the message could happen to be an AssertError (or something else) instead of a string. I know it because Touraj found a bug for import:file, which was failing also in node-cli (which we are calling under the hood) and we had an error when calling message.include() function from this logger, so I just want to make a sanity check before calling it.